openpype/plugins/publish/extract_review_slate.py

```diff
@@ -28,7 +28,26 @@ class ExtractReviewSlate(publish.Extractor):
 
     SUFFIX = "_slate"
 
-    hosts = ["nuke", "shell"]
+    hosts = [
+        "nuke",
+        "maya",
+        "blender",
+        "houdini",
+        "shell",
+        "hiero",
+        "premiere",
+        "harmony",
+        "traypublisher",
+        "standalonepublisher",
+        "fusion",
+        "tvpaint",
+        "resolve",
+        "webpublisher",
+        "aftereffects",
+        "flame",
+        "unreal"
+    ]
+
     optional = True
```

Comment:
Unrelated to this PR butl ikely a Nuke bug. The optional state here doesn't actually work in the new publisher. Right? @iLLiCiTiT

Reply:
Did you already created issue or should I create it?